megablocks/layers/gelu.py

```diff
@@ -1,7 +1,72 @@
 import stk
+import torch
 import torch.nn.functional as F
 
-def gelu(x):
+
+@torch.jit.script
+def _gelu_forward(x):
+    return  x * 0.5 * (1.0 + torch.tanh(0.79788456 * x * (1 + 0.044715 * x * x)))
```

Comment:
U could probably just use 
```
with torch.no_grad():
    return F.gelu(x, approximate="tanh")
```
here instead.

Reply:
wrap in `with torch.no_grad():` ctx manager if needed.

Reply:
Will replace - originally put in to modify to be in-place, but not needed.